content/pt-br/docs/concepts/overview/working-with-objects/namespaces.md

```diff
@@ -0,0 +1,106 @@
+
+## Trabalhando com Namespaces
+
+Criação e eliminação de namespaces estão descritas na 
+[documentação de namespaces do guia de administradores](/docs/tasks/administer-cluster/namespaces).
+
+{{< note >}}
+    Evite criar namespaces com o prefixo `kube-`, já que este prefixo é reservado para namespaces do sistema Kubernetes.
+{{< /note >}}
+
+### Visualizando namespaces
+
+Você pode obter uma lista dos namespaces atuais dentro de um cluster com:
+
+```shell
+kubectl get namespace
+```
+```
+NAME              STATUS   AGE
+default           Active   1d
+kube-node-lease   Active   1d
+kube-public       Active   1d
+kube-system       Active   1d
+```
+
+Kubernetes se inicializa com quatro namespaces iniciais:
+
+   * `default` O namespace padrão para objetos sem namespace
+   * `kube-system` O namespace para objetos criados pelo sistema Kubernetes
+   * `kube-public` Este namespace é criado automaticamente e é legível por todos os usuários (incluindo usuários não autenticados). Este namespace é reservado principalmente para uso do cluster, no caso de alguns recursos que precisem ser visíveis e legíveis publicamente por todo o cluster. O aspecto público deste namespace é apenas uma convenção, não um requisito.
+   * `kube-node-lease` Este namespace contém os objetos de [Lease](/docs/reference/kubernetes-api/cluster-resources/lease-v1/) associados com cada node. Node leases permitem que o kubelet envie [heartbeats](/docs/concepts/architecture/nodes/#heartbeats) para que a camada de gerenciamento detecte falhas nos nodes.
+
+### Preparando o namespace para uma requisição
+
+Para preparar o namespace para a requisição atual, utilize o parâmetro `--namespace`. Por exemplo:
+
+```shell
+kubectl run nginx --image=nginx --namespace=<insert-namespace-name-here>
+kubectl get pods --namespace=<insert-namespace-name-here>
+```
+
+### Configurando a preferência de namespaces
+
+Você pode salvar permanentemente o namespace para todos os comandos `kubectl` subsequentes no mesmo contexto:
+
+```shell
+kubectl config set-context --current --namespace=<insert-namespace-name-here>
+# Validando
+kubectl config view --minify | grep namespace:
+```
+
+## Namespaces e DNS
+
+Quando você cria um [Serviço](/docs/concepts/services-networking/service/), ele cria uma
+[entrada DNS](/docs/concepts/services-networking/dns-pod-service/) correspondente.
+Esta entrada possui o formato: `<service-name>.<namespace-name>.svc.cluster.local`, de forma que se um container utilizar apenas `<service-name>` ele será resolvido para um serviço que é local ao namespace.
+Isso é útil para utilizar a mesma configuração em vários namespaces, por exemplo em Desenvolvimento, `Staging` e `Production`. Se você quiser acessar múltiplos namespaces, precisará utilizar um _Fully Qualified Domain Name_ (FQDN).
```

Comment:
Here you can translate Development and Production and use Staging (I don't find any Portuguese work to translate that)